docs/maui/converters/string-to-list-converter.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
@brminnick lets try that again, this is the file I would like you to take a look over given the more complex markup scenario.